app/models/concerns/resource.rb

```diff
@@ -71,7 +71,7 @@ def resource_name
       resource_name_prefix
     end
 
-    def filter(conditions)
+    def filter_with(conditions)
```

Comment:
What's the point of a single-use method which is just an alias for `where`?

Reply:
:shrug: